packages/expo-modules-core/ExpoModulesCore.podspec

```diff
@@ -114,6 +114,9 @@ Pod::Spec.new do |s|
 
   s.test_spec 'Tests' do |test_spec|
     test_spec.dependency 'ExpoModulesTestCore'
+    if reactNativeTargetVersion >= 77
```

Comment:
we don't need this version check. i believe we don't support 0.76 or earlier after this pr